src/ota.html

```diff
@@ -122,10 +122,10 @@ <h3>Outputs</h3>
         <dd>
             Satus message of the current progress. It is one of the following messages:
             <ul>
-                <li><code>Update started</code> will be sent when the update starts (Output 1 - "start/end"</li>
-                <li><code>Update succeded</code> will be sent when the update succeded (Output 1 - "start/end")</li>
+                <li><code>Update started</code> will be sent when the update starts (Output 1 - "start/end")</li>
+                <li><code>Update succeded</code> will be sent when the update succeeded (Output 1 - "start/end")</li>
```

Comment:
In theory, you could change this also to "succeeded" - it's not wrong, just not so common I would say.

Reply:
sorry, i dont understand what you mean

Reply:
Ah, highlighting does not work as in azure DevOps.
In the code part it's spelled differently: `<code>Update succeded</code> `

Reply:
ah thanks, changed it